src/linux/linux_platform.cpp

```
@@ -384,6 +384,49 @@ std::optional<int32_t> resolve_builtin_call_linux(const std::string& name) {
+    // is responsible for mapping to target-kernel/module IDs in production use.
+    if (name == "bpf_skb_ct_lookup") {
+        return KsymBtfId{.btf_id = 20001, .module = 0};
+    }
+    if (name == "bpf_ct_release") {
+        return KsymBtfId{.btf_id = 20002, .module = 0};
+    }
+    if (name == "bpf_fentry_test1") {
+        return KsymBtfId{.btf_id = 20003, .module = 0};
+    }
+    if (name == "bpf_cpumask_create") {
+        return KsymBtfId{.btf_id = 1009, .module = 0};
```

Cppcheck (2.19.0)

[style] The function 'abs' is never used.

(unusedFunction)